newrelic/hooks/mlmodel_openai.py

```diff
@@ -90,6 +90,11 @@ def wrap_chat_completion_sync(wrapped, instance, args, kwargs):
     completion_id = str(uuid.uuid4())
     request_message_list = kwargs.get("messages", [])
 
+    # If `.with_streaming_response.` wrapper used, switch to streaming
```

Comment:
Same thing here, let's move this up to the if not transaction early exit so we don't record those attributes on the transaction before exiting.